beacon-chain/rpc/eth/light-client/handlers.go

```diff
@@ -46,18 +46,52 @@ func (s *Server) GetLightClientBootstrap(w http.ResponseWriter, req *http.Reques
 		return
 	}
 
-	bootstrap, err := createLightClientBootstrap(ctx, state)
-	if err != nil {
-		httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+	ver := blk.Version()
+	if ver >= version.Deneb {
```

Comment:
It would be better to do a `switch` statement and return an error if version is other than Altair, Capella or Deneb